Makefile

```diff
@@ -65,7 +65,7 @@ container-image:
 		--build-arg HUGO_VERSION=$(HUGO_VERSION)
 
 container-build: module-check
-	$(CONTAINER_RUN) --read-only $(CONTAINER_IMAGE) hugo --minify
+	$(CONTAINER_RUN) --read-only $(CONTAINER_IMAGE) sh -c "npm ci && hugo --minify"
```

Comment:
I am testing these changes now. Ran into a Go panic. Investigating:

```
added 120 packages in 3.021s
Total in 2 ms
panic: mkdir /tmp/hugo_cache: read-only file system

```

Reply:
As an aside, the module check initially failed for me.
I needed to update the theme submodule. The script warned of the problem, though,
could add a dependent target to run,
`git submodule update --init --recursive --depth 1`

Reply:
Hmm. I made some Docker changes recently. The current `Makefile` is supposed to run the container with a tmpfs mount for `/tmp`
Maybe that's also needed for `container-build` as well as `container-serve`.

Reply:
When is the  `container-build` target run?
The Dockerfile has `npm install ... autoprefixer, postcss-cli` in the `apk add` command.

Reply:
`make container-image` updates the container image
`make container-build` runs a website build, using Hugo in a container
`make container-serve` runs a website build to a temporary folder and then serves it at http://localhost:1313/, also using Hugo and also in a container

Reply:
I don't ever run `make container-build`. If you wanted to run your own checks on the generated HTML, that's when you might run it.

Reply:
OK. I did run the `container-build` target previously.
@inductor , I'm confused. Does the target work without the addition of "npm ci"?
Do you have untracked files after the target is run?
If so, I suggest to add these directories to the .gitignore file?
I am investigating why the `container-build` target does not work (writing to the hugo_cache directory) for me.

Reply:
Hi. I made some progress in testing the npm changes.
I created a new clone of the site. Updated `npm`.
I was able to get the original `container-build` target to run though the CSS is not generated,

```
Error: Error building site: POSTCSS: failed to transform "scss/main.css resource "scss/scss/main.scss_4853eb546e7a6c0898ed71feae7357c0" not found in file cache"
```

I then tested the proposed changes and the command hangs. I removed the `read-only` flag
and the command progresses a bit more.
You need to have at least version 10 of node and 9.0.0 of the `fs-extra` library.
There is an issue logged with the theme (https://github.com/google/docsy/issues/265).
Still working on a clean build.
Changed the command to:
```
$(CONTAINER_RUN)  $(CONTAINER_IMAGE) sh -c "npm install -D autoprefixer && npm install -D postcss-cli && hugo --minify"
```

Reply:
FYI: I'm facing POSTCSS errors too but resolved by the following command.

```shell
container-build: module-check
        $(CONTAINER_RUN) --read-only --mount type=tmpfs,destination=/tmp,tmpfs-mode=01777 $(CONTAINER_IMAGE) sh -c "npm install -D autoprefixer postcss postcss-cli && hugo --minify"
```

Reply:
Thanks @makocchi-git . Yes, I suspect it will.  This change would get things working again.